shell.nix

```diff
@@ -0,0 +1,10 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+pkgs.mkShell {
+  buildInputs = with pkgs; [
+    go goimports gopls
+
+    # keep this line if you use bash
+    pkgs.bashInteractive
```

Comment:
Why? I've never included shells in my shell.nix, and it just inherits from its environment. What do we gain from this?

Reply:
This was something lorri added for me. I'll remove it.